pkg/scheduler/preemption/preemption_test.go

```diff
@@ -1522,6 +1522,83 @@ func TestFairPreemptions(t *testing.T) {
 			targetCQ:      "a",
 			wantPreempted: sets.New("/a_low", "/b1"),
 		},
+		"preempt huge workload if there is no other option, as long as the target CQ gets a lower share": {
+			admitted: []kueue.Workload{
+				*utiltesting.MakeWorkload("b1", "").Request(corev1.ResourceCPU, "9").SimpleReserveQuota("b", "default", now).Obj(),
+			},
+			incoming:      utiltesting.MakeWorkload("a_incoming", "").Request(corev1.ResourceCPU, "2").Obj(),
+			targetCQ:      "a",
+			wantPreempted: sets.New("/b1"),
+		},
+		"can't preempt huge workload if the incoming is also huge": {
+			admitted: []kueue.Workload{
+				*utiltesting.MakeWorkload("a1", "").Request(corev1.ResourceCPU, "2").SimpleReserveQuota("a", "default", now).Obj(),
+				*utiltesting.MakeWorkload("b1", "").Request(corev1.ResourceCPU, "9").SimpleReserveQuota("b", "default", now).Obj(),
```

Comment:
```suggestion
				*utiltesting.MakeWorkload("a1", "").Request(corev1.ResourceCPU, "2").SimpleReserveQuota("a", "default", now).Obj(),
				*utiltesting.MakeWorkload("b1", "").Request(corev1.ResourceCPU, "7").SimpleReserveQuota("b", "default", now).Obj(),
```

Is this correct? Because the sum of quota is `9` (3+3+3), I guess that `a1` or `b1` are not admitted, right?

Reply:
That's correct. Also changed the incoming to 5, just to be at the limit.